helper/helper_test.go

```diff
@@ -504,20 +575,84 @@ var _ = Describe("helper tests", func() {
 			Expect(err).To(HaveOccurred())
 			Expect(err.Error()).To(Equal(msg))
 		})
+		It("CopyData, readerType is SUBSET. Normal completion", func() {
+			writer = bufio.NewWriterSize(&limitWriter{100}, 5)
```

Comment:
Shouldn't this be reset before each test so that the setup doesn't affect the rest of the tests?

Reply:
or maybe reset after each?